src/app/components/MostRead/Canonical/Item/index.tsx

```diff
@@ -47,13 +50,15 @@ export const MostReadLink = ({
   eventTrackingData,
 }: PropsWithChildren<MostReadLinkProps>) => {
   const clickTrackerHandler = useClickTrackerHandler(eventTrackingData);
+  const liteUrl = constructLiteSiteURL(eventTrackingData);
 
   return (
     <div css={getItemCss({ dir, size })} dir={dir}>
       <a
         css={[styles.link, size === 'default' && styles.defaultLink]}
         href={href}
         onClick={clickTrackerHandler}
+        {...(liteUrl && {'add-lite-tracker-params': liteUrl})}
```

Comment:
Should we make 'add-lite-tracker-params' into a const somewhere which we can reuse in other places?
```suggestion
        {...(liteUrl && {LITE_TRACKER_PARAM: liteUrl})}
```